pytket/pytket/circuit/display/__init__.py

```diff
@@ -173,21 +246,30 @@ def render_circuit_as_html(
             return None
         return html
 
-    def render_circuit_jupyter(self, circuit: RenderCircuit) -> None:
+    def render_circuit_jupyter(
+        self,
+        circuit: RenderCircuit | List[RenderCircuit],
+        orient: Orientation | None = None,
+    ) -> None:
         """Render a circuit as jupyter cell output.
 
-        :param circuit: the circuit to render.
+        :param circuit: the circuit(s) to render.
+        :param orient: the direction in which to stack circuits if multiple are present.
```

Comment:
Should say here what the values of `orient` can be.

Reply:
Does the `orient` value just dictate whether multi circuit rendering stacks the circuits on top of one another or be arranged next to each other in a row?

Reply:
yes - you can also set a default for it but seems like something users are more likely to change depending on the circuits to be displayed